arcgis-ios-sdk-samples/Layers/Display WFS/DisplayWFSViewController.swift

```diff
@@ -0,0 +1,98 @@
+// Copyright 2019 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class DisplayWFSViewController: UIViewController {
+
```

Comment:
Please remove this empty line. It generates a warning for violating the [Vertical Whitespace after Opening Braces](
https://github.com/realm/SwiftLint/blob/master/Rules.md#vertical-whitespace-after-opening-braces) linting rule.

![Screen Shot 2019-03-20 at 3 59 46 PM](https://user-images.githubusercontent.com/2257493/54724695-74648d80-4b29-11e9-80f4-8a2f47bbe697.png)